nectar/sample-config.toml

```diff
@@ -63,3 +45,21 @@ node_url = "http://localhost:18443/"
 chain_id = 1
 # The url to the web3 node, can include an infura key: `https://mainnet.infura.io/v3/YOUR-PROJECT-ID`
 node_url = "http://localhost:8545/"
+
+# Strategies used for Bitcoin fee handling.
+[fee_strategies.bitcoin]
```

Comment:
This could also just be called `[fees.bitcoin]`.

Reply:
We also already have a `bitcoin` section, could possibly also have `[bitcoin.fees]`? That would "repeat" the `fees` section with every ledger but at the same time would allow us to use ledger-specific terminology and for example do `[ethereum.gas]` instead.

Generally, I feel like we should strive to design things per-ledger and avoid having to many abstractions like "fees" in this case that span multiple ledgers.